ext/intl/php_intl.stub.php

```diff
@@ -17,9 +17,9 @@ function intlcal_get_now(): float {}
 
 function intlcal_get_available_locales(): array {}
 
-function intlcal_get(IntlCalendar $calendar, int $field): int {}
+function intlcal_get(IntlCalendar $calendar, int $field): int|false {}
```

Comment:
As these were missing from the free-standing functions, does that mean there is no test coverage for them? Or is the test coverage on the OO methods?

Or to ask the question another way, can all these false values really occur or it a matter of them being theoretically possible, as an error check happens?

Reply:
Based on https://github.com/unicode-org/icu/blob/4c35e59bdc06341aa8a8c8f50bfb3ea1074c5ed3/icu4c/source/i18n/calendar.cpp#L1440 this happens for "buffer overflows". Don't know if possible to trigger from PHP code, but I guess we need to assume it is.